### PR TITLE
Added configuration to change snippets according to where the user are

### PR DESCRIPTION
**Pull request by zLianK**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,6 +97,15 @@
 					"type": "boolean",
 					"description": "Controls whether Rech Batch extension should alert when conflicting batch extensions are detected.",
 					"default": true
+				},
+				"rech.batch.snippetsType": {
+					"type": "string",
+					"default": "Community",
+					"description": "Switch to different snippets (you should use the community configuration if you are not at Rech)",
```
Comment (gustavosimon) [inline package.json:104]: I don't know if it is properly to say that the user should use the community setting if he's not at Rech.

User may like Rech's snippets and use it if he can understand portuguese language.

@zLianK @Lucasspaniol What do you think about it?

File: snippets/bat-community.json [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    },
+    "ifn": {
+        "prefix": "ifn",
+        "body": "if not \"%$1%\"==\"$2\" (\n   $3\n) else (\n   $4\n)",
+        "description": "Check negative equality"
+    },
+    "ifne": {
+        "prefix": "ifne",
+        "body": "if not exist \"$1\" (\n   $2\n) else (\n   $3\n)",
+        "description": "Check if a file not exists"
```
Comment (gustavosimon) [inline snippets/bat-community.json:60]: Please replace by "Check if a file does not exists"

**Review by gustavosimon — CHANGES_REQUESTED** [review]
LGTM

I think we can create a new improvement issue to change source file of the snippets. So, the user can add custom snippets by yourself.

File: package.json (hunk 1 above)
Comment (zLianK) [reply package.json:104]: I agree with you, let me change this literal.

**Review by gustavosimon — APPROVED** [review]

**Comment by gustavosimon** [comment]
This PR fixes #13.

> I think we can create a new improvement issue to change source file of the snippets. So, the user can add custom snippets by yourself.

@zLianK Can you create this issue?

**Review by Lucasspaniol — APPROVED** [review]
😎

**Pull request merged**